src/util/config.rs

```diff
@@ -739,6 +743,142 @@ pub fn check_kernel() -> Vec<ConfigError> {
     Vec::new()
 }
 
+#[derive(Debug, Default)]
+struct FsInfo {
+    tp: String,
+    opts: String,
+    mnt_dir: String,
+    fsname: String,
+}
+
+#[cfg(target_os = "linux")]
+fn get_fs_info(path: &str, mnt_file: &str) -> Result<FsInfo, ConfigError> {
```

Comment:
Put things into module. Otherwise it becomes unused function on non-linux platform.